src/components/Form/CreateProfileForm/ProfileData/ProfileData.tsx

```diff
@@ -61,7 +63,9 @@ const defaultInitialValues = {
   description: "",
   description_en: "",
   description_de: "-",
-  salesChannel: defaultSalesChannel
+  salesChannel: defaultSalesChannel,
+  supportType: "",
+  supportValue: ""
```

Comment:
What do you think to add a default value for the field `supportValue` an empty string instead of `-----` at row `251` into Formik initialValues of the `src/components/Form/CreateProfileForm/ProfileData/ProfileData.tsx` component?

Reply:
I left what i found, but i think it's better to change to empty string